CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## [1.0.0] - 2021-02-02
+
+### Added
+- Filtering by federated trust domains in `ListEntries` (#1967)
+- Logging when lookup of user by uid or group by gid fails in the `unix` WorkloadAttestor plugin (#2048)
+- Timeouts to calls to KeyManager plugins (#2044)
+- gRPC Health v1 services in Server and Agent exposed at `/grpc.health.v1.Health/Check` path (#2057, #2058, #2087)
```

Comment:
nit: lowercase "server" and "agent" when not preceded by SPIRE. There are other instances of this.